.edm4hep-ci.d/compile_and_test_with_podio.sh

```diff
@@ -12,5 +12,6 @@ cmake -DCMAKE_CXX_STANDARD=17 -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=../inst
 cd ..
 export CMAKE_PREFIX_PATH=$PWD/install:$CMAKE_PREFIX_PATH
 export PODIO=$PWD/install:$CMAKE_PREFIX_PATH
+export LD_LIBRARY_PATH=$PWD/install/lib:$LD_LIBRARY_PATH
```

Comment:
@vvolkl: The files are installed to lib64
https://travis-ci.com/github/key4hep/EDM4hep/jobs/385056062#L505